data/campaigns/Heir_To_The_Throne/scenarios/20b_Underground_Channels.cfg

```diff
@@ -430,7 +430,7 @@
                 [/message]
                 [message]
                     speaker=unit
-                    message= _ "Then you should look to the north and east. But, if you would, might we ask your assistance."
+                    message= _ "Then you should look to the north and east. But, if you would, might we ask your assistance?"
```

Comment:
I think this should go in the po-fixes script? Assuming that's actually used...

Reply:
Is that something I should be concerning myself with?

Reply:
If I understand correctly it's generally supposed to be the responsibility of people who correct typos and similar errors to add the change to po-fixes. Presumably @ivanovic can confirm/deny?